apisix/plugins/kafka-logger.lua

```diff
@@ -51,6 +52,26 @@ local schema = {
                 },
             },
         },
+        brokers = {
```

Comment:
Please mark the `broker_list` in the schema as deprecated.

Reply:
Fixed.